### PR TITLE
feat: FED Recommends

### PR DESCRIPTION
**Pull request by alvinhui**
https://github.com/ice-lab/iceworks/issues/821

## 依据

- 流行：符合业界成熟的前端技术栈
- 稳定：最近依然保持更新、知名作者或有赞助商的支持
- 可靠：下载量较高、被 Pack 引用的次数多
- 体验：使用方式简单、文档质量高

## 入选

- [Chinese (Simplified) Language Pack](https://marketplace.visualstudio.com/items?itemName=MS-CEINTL.vscode-language-pack-zh-hans)：Language pack extension for Chinese (Simplified)
- [GitLens](https://marketplace.visualstudio.com/items?itemName=eamodio.gitlens): Supercharge the Git capabilities built into Visual Studio Code
- [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker): Spelling checker for source code
- [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint): Integrates [ESLint](https://eslint.org/) JavaScript into VS Code
- [stylelint](https://marketplace.visualstudio.com/items?itemName=stylelint.vscode-stylelint): Lint CSS/SCSS/Less with [stylelint](https://stylelint.io/)
- [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode): An opinionated code formatter. It enforces a consistent style by parsing your code and re-printing it with its own rules that take the maximum line length into account, wrapping code when necessary.
- [Auto Complete Tag](https://marketplace.visualstudio.com/items?itemName=formulahendry.auto-complete-tag): Combine the functions of Auto Close Tag and Auto Rename Tag
- [JavaScript (ES6) code snippets](https://marketplace.visualstudio.com/items?itemName=xabikos.JavaScriptSnippets): Code snippets for JavaScript in ES6 syntax
- [Simple React Snippets](https://marketplace.visualstudio.com/items?itemName=burkeholland.simple-react-snippets): The essential collection of React Snippets and commands.
- [Debugger for Chrome](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome): Debug your JavaScript code in the Chrome browser, or any other target that supports the Chrome Debugger protocol.
- [Bookmarks](https://marketplace.visualstudio.com/items?itemName=alefragnani.Bookmarks): It helps you to navigate in your code, moving between important positions easily and quickly
- [Iceworks](https://marketplace.visualstudio.com/items?itemName=iceworks-team.iceworks): Visual Intelligent Development Pack, provide visualization and intelligent technology to build Universal Application faster and better

### CODE REVIEW

File: extensions/fed/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+[![Downloads](https://vsmarketplacebadge.apphb.com/downloads-short/iceworks-team.fed.svg)](https://marketplace.visualstudio.com/items?itemName=iceworks-team.fed)
+[![Rating](https://vsmarketplacebadge.apphb.com/rating-star/iceworks-team.fed.svg)](https://marketplace.visualstudio.com/items?itemName=iceworks-team.fed&ssr=false#review-details)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/ice-lab/iceworks/pulls)
+[![The MIT License](https://img.shields.io/badge/license-MIT-blue.svg)](http://opensource.org/licenses/MIT)
+
+Recommend Extensions for Development by [FED Team](https://fed.taobao.org/).
+
+Based on：
+
+- In line with the popular, mature front-end technology stack
```
Comment (fyangstudio) [inline extensions/fed/README.md:16]: 统一格式？
Popular：
High stability:
High reliability:
Good experience:

Comment (alvinhui) [reply extensions/fed/README.md]: done

**Comment by imsobear** [comment]
- React 领域有必备的插件吗？
- Iceworks 要加进去？套娃？
- 品牌上 FED 是否合适？F2E？

**Comment by alvinhui** [comment]
> * React 领域有必备的插件吗？
> * Iceworks 要加进去？套娃？
> * 品牌上 FED 是否合适？F2E？

1. React 必备的插件从 VS Code 插件市场中比较[流行的 React Pack](https://marketplace.visualstudio.com/search?term=react&target=VSCode&category=Extension%20Packs&sortBy=Installs) 来看，我们这里面的列表已经覆盖了一部分
2. 我理解这里的定位是。是否合理大家可以探讨哈
    - FED: 前端必备 VS Code 插件集合
    - Iceworks: 我们自研的多端研发套件
3. 这个我的想法是代表 FED 的最佳实践，有点品牌背书的味道，当前我们是否能广泛地代表 FED 有待商榷

**Review by fyangstudio — APPROVED** [review]

**Pull request closed without merging**